storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2212,10 +2314,6 @@ dberr_t dict_load_tables_from_space_id(space_id_t space_id, THD *thd,
                                   compressed_sdi_len);
     decompressor.decompress();
 
-    if (ib_key.sdi_key->type != 1 /* dd::Sdi_type::TABLE */) {
-      continue;
-    }
-
```

Comment:
why did you remove this? we are interested on in TABLE type records right?

Reply:
I moved this little above few lines to avoid some more computation if sdi_key->type is 1